src/components/Note/Note.js

```diff
@@ -19,11 +19,16 @@ const propTypes = {
   annotation: PropTypes.object.isRequired,
 };
 
+const RETURN_FALSE = () => false;
+
 const Note = ({ annotation }) => {
   const { isSelected, resize } = useContext(NoteContext);
   const containerRef = useRef();
   const containerHeightRef = useRef();
 
+  const isReplyDisabledFunc = useSelector(state => selectors.getIsReplyDisabled(state)) || RETURN_FALSE;
+  const isReplyDisabled = isReplyDisabledFunc(annotation);
```

Comment:
I think you can just do something like `isReplyDisabledFunc?.(annotation)`. 
It's equivalent to
```javascript
const isReplyDisabled = isReplyDisabledFunc ? isReplyDisabledFunc(annotation) : undefined;
```
This way you don't need to have the `RETURN_FALSE` function.